api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -54,57 +50,67 @@
  * It is a default implementation of {@link RegistrationCoreService}.
  */
 @Transactional
-public class RegistrationCoreServiceImpl extends BaseOpenmrsService implements RegistrationCoreService, GlobalPropertyListener, ApplicationContextAware {
-	
+public class RegistrationCoreServiceImpl extends BaseOpenmrsService implements RegistrationCoreService, ApplicationContextAware {
+
 	protected final Log log = LogFactory.getLog(this.getClass());
-	
+
 	private ApplicationContext applicationContext;
-	
+
 	private RegistrationCoreDAO dao;
-	
+
 	private PatientService patientService;
-	
+
 	private PersonService personService;
-	
-	private LocationService locationService;
-	
+
 	private AdministrationService adminService;
-	
-	private static IdentifierSource idSource;
-	
+
+	private IdentifierBuilder identifierBuilder;
+
+	private MatchedPatientFilter matchedPatientFilter;
+
+	private RegistrationCoreProperties coreProperties;
+
 	@Override
 	public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
 		this.applicationContext = applicationContext;
 	}
-	
+
 	public void setDao(RegistrationCoreDAO dao) {
 		this.dao = dao;
 	}
-	
+
 	/**
 	 * @return the dao
 	 */
 	public RegistrationCoreDAO getDao() {
 		return dao;
 	}
-	
+
 	public void setPatientService(PatientService patientService) {
 		this.patientService = patientService;
 	}
-	
+
 	public void setPersonService(PersonService personService) {
 		this.personService = personService;
 	}
-	
-	public void setLocationService(LocationService locationService) {
-		this.locationService = locationService;
-	}
-	
+
 	public void setAdminService(AdministrationService adminService) {
 		this.adminService = adminService;
 	}
 
-    /**
+	public void setIdentifierBuilder(IdentifierBuilder identifierBuilder) {
+		this.identifierBuilder = identifierBuilder;
+	}
+
+	public void setMatchedPatientFilter(MatchedPatientFilter matchedPatientFilter) {
+		this.matchedPatientFilter = matchedPatientFilter;
+	}
+
+	public void setCoreProperties(RegistrationCoreProperties coreProperties) {
+		this.coreProperties = coreProperties;
```

Comment:
This property should be called either "moduleProperties" or "registrationCoreProperties". (coreProperties is misleading because that makes one think that there are some other extended properties somewhere else.)